details.html

```diff
@@ -152,434 +156,764 @@
+                        calendar_month
+                      </span>
+                    </div>
 
-                      <img
-                        src="./images/user3.png"
-                        class="avatar round small"
-                        alt="Shavkat Mirzoev avatar"
-                      />
+                    <div>
+                      <span class="title text-ellipsis">Start Date</span>
+                      <span class="subtitle text-ellipsis">17 Jun, 2020</span>
                     </div>
+                  </div>
 
-                    <div class="badge large danger-light icon">
-                      <span class="material-icons-outlined icon">
-                        watch_later
+                  <div class="stat-info">
+                    <div class="badge large icon-only warning light">
+                      <span class="material-icons-round icon">
+                        calendar_month
                       </span>
+                    </div>
 
-                      <span>2 days left</span>
+                    <div>
+                      <span class="title text-ellipsis">End Date</span>
+                      <span class="subtitle text-ellipsis">04 Jul, 2020</span>
                     </div>
-                  </footer>
+                  </div>
+                </div>
+              </section>
+
+              <section class="description">
+                <h2 class="content-title">description</h2>
+                <p>
+                  You need to develop an application on something like React
+                  native, so that it is for Android and IOS. There are about 30
+                  screens, the design and layout in the sketch is ready. The
+                  main pages are login, getting a task, a list of tasks, a map,
+                  a history of tasks, calling the camera to complete a task. The
+                  storage and processing server is on our side, there is a
+                  ready-made api for the web service that you will need to use.
+                </p>
+              </section>
+
+              <section class="checklist">
+                <h2 class="content-title">checklist (50%)</h2>
+
+                <div class="progress-bar primary">
+                  <div class="progress" style="width: 50%"></div>
                 </div>
-              </li>
 
-              <li>
-                <div class="related-projects-list-item">
-                  <a href="#" class="mini-profile">
-                    <img
-                      src="./images/vue.jpg"
-                      alt="V shaped grey and turquoise lines"
-                      class="avatar large"
-                    />
+                <div class="tasks list-unstyled">
+                  <li>
+                    <div role="button" class="task completed">
+                      <span class="material-icons icon">task_alt</span>
+                      <span class="task-txt"><s>Create wireframes</s></span>
```

Comment:
1. What is `txt`? I don't understand that language.
2. Why you decided to go with `s` tag? We discussed it and css way was better.